content/en/monitors/guide/_index.md

```diff
@@ -40,6 +40,7 @@ cascade:
     {{< nextlink href="monitors/guide/why-did-my-monitor-settings-change-not-take-effect" >}}Monitor settings changes not taking effect{{< /nextlink >}}
     {{< nextlink href="monitors/guide/integrate-monitors-with-statuspage" >}}Integrate monitors with Statuspage{{< /nextlink >}}
     {{< nextlink href="monitors/guide/github_gating" >}}Gating your GitHub Actions Deployments with Datadog Monitors{{< /nextlink >}}
+    {{< nextlink href="monitors/guide/change-alert" >}}Change Alert monitors{{< /nextlink >}}
```

Comment:
Optional suggestion here, but I think grouping this with the `Anomaly monitors` entry makes slightly more sense for organization